### ISSUE TITLE
Airspeed not available on stream deck plus

### ISSUE THREAD

**Issue by anton264** [nguyenquyhy/Flight-Tracker-StreamDeck#404]
Below are some useful information that you can include to help with troubleshooting the issue:
- Stream Deck Plugin and MSFS version
  **0.51** version of stream deck plugin  

In `PresetChangeAction.cs` on row 105 there seems to be an option for airspeed to be set, however in the plugin there is no option for airspeed. This also applies to QNH

![image](https://github.com/nguyenquyhy/Flight-Tracker-StreamDeck/assets/10194266/ac595338-afdf-4aed-876b-af41f9da9e63)

**Comment by nguyenquyhy** [comment]
Have you tried to use FLC option?

**Comment by anton264** [comment]
Yes the FLC option works on aircraft that has FLC. But on aircraft that does not have FLC, such as helicopters there is no way to adjust airspeed with the dials.

**Comment by anton264** [comment]
I'm using the hype performance Airbus H145
